easycv/models/detection/detectors/yolox/yolo_pafpn.py

```diff
@@ -1,90 +1,281 @@
 # Copyright (c) 2014-2021 Megvii Inc And Alibaba PAI-Teams. All rights reserved.
+import logging
+import math
 
 import torch
 import torch.nn as nn
 
 from easycv.models.backbones.darknet import CSPDarknet
-from easycv.models.backbones.network_blocks import BaseConv, CSPLayer, DWConv
+from easycv.models.backbones.network_blocks import (BaseConv, CSPLayer, DWConv,
+                                                    GSConv, VoVGSCSP)
+from easycv.models.backbones.repvgg_yolox_backbone import RepVGGYOLOX
+
+
+def make_divisible(x, divisor):
```

Comment:
merge and reuse repvgg_yolox_backbone and yolo_pafpn, reuse op move to model/utils

Reply:
done